README.md

```diff
@@ -276,6 +281,82 @@ TurkishCities.find_by_postcode(34382)    # => "Couldn't find any subdistrict wit
 TurkishCities.find_by_postcode('100000') # => Given value [100000] is outside bounds of 1010 to 81952.
 ```
 
+### Finding city name with population data
+
+### Exact search
+
+Search city name with exact population
+
+```rb
+TurkishCities.find_by_population('exact', 15840900)
```

Comment:
I think we don't need three working examples here a single example would be enough. This applies to other newly added methods as well (`below`, `above`, `between`)